addons/website/static/src/client_actions/configurator/configurator.js

```diff
@@ -82,6 +84,33 @@ export const PALETTE_NAMES = [
 // from CSS and added in each palette.
 export const CUSTOM_BG_COLOR_ATTRS = ["menu", "footer"];
 
+const NBR_MAIN_THEMES = 3;
+const NBR_EXTRA_THEMES = 24;
+
+/**
+ * Returns a list of maximum "resultNbrMax" themes that depends on the wanted
+ * industry and the color palette.
+ *
+ * @param {Object} orm - The orm used for the server call.
+ * @param {Number} industryId - The id of the wanted industry.
+ * @param {Object} palette - The color palette to use in the svgs.
+ * @param {Number} resultNbrMax - The number of different wanted themes.
+ * @returns {Promise<Array>} A list of objects that contains the different
+ * theme names and their related text svgs (as result of a Promise). The length
+ * of the list is at most 'resultNbrMax'.
+ */
+async function getRecommendedThemes(orm, industryId, palette, resultNbrMax=NBR_MAIN_THEMES) {
```

Comment:
Like this, I agree that this function has low value (why not call the ORM directly repeating the python function). To have value, pass the whole state instead of industryId and palette. Then you can read the state and then this function has value: it reads the correct entry from the state and call the python function accordingly.